backend/prompt_studio/prompt_studio_core/models.py

```diff
@@ -101,3 +116,11 @@ class CustomTool(BaseModel):
     enable_challenge = models.BooleanField(
         db_comment="Flag to enable or disable challenge", default=False
     )
+
+    # Introduced field to establish M2M relation between users and custom_tool.
+    # This will introduce intermediary table which relates both the models.
+    shared_users = models.ManyToManyField(
```

Comment:
@johnyrahul What will the value of `shared_users` be for Prompt projects created before this field was introduced?

Reply:
@harini-venkataraman  when we introduce this relation it creates an intermediary table.

Attached screenshot for better understanding
![image](https://github.com/Zipstack/unstract/assets/116638720/2fe786bc-53d0-4620-86da-dba29a0e86ac)